ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -325,6 +376,18 @@ func deleteCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	return nil
 }
+func deleteEmptyClusterPrompt(reader *bufio.Reader, cliParams *config.CLIParams) error {
+	fmt.Printf("CloudFormation stack not found for cluster '%s'. Are you sure you want to proceed? [y/N]\n", cliParams.Cluster)
+	input, err := reader.ReadString('\n')
+	if err != nil {
+		return fmt.Errorf("Error reading input: %s", err.Error())
+	}
+	formattedInput := strings.ToLower(strings.TrimSpace(input))
+	if formattedInput != "yes" && formattedInput != "y" {
+		return fmt.Errorf("Aborted cluster deletion. To delete your cluster, re-run this command and specify the '--%s' flag or confirm that you'd like to delete your cluster at the prompt.", flags.ForceFlag)
```

Comment:
Nit: Personally I think this message should be printed at the Info level and that responding no should not be an error case. If the user doesn't confirm, the CLI could just immediately do `os.Exit(0)`.

Reply:
```
$ ecs-cli down
Are you sure you want to delete your cluster? [y/N]
n
FATA[0001] Error executing 'down': Aborted cluster deletion. To delete your cluster, re-run this command and specify the '--force' flag or confirm that you'd like to delete your cluster at the prompt. 
```

I know that the old prompt works the same as yours, but this PR has made me think about it and I don't really like how failing to confirm at the prompt prints a Fatal warning, as if something unexpected has happened.